src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/updatecontactdetails/UpdateContactDetailsAboutToSubmitHandler.java

```diff
@@ -100,4 +93,25 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
         return GenericAboutToStartOrSubmitCallbackResponse.<FinremCaseData>builder()
             .data(finremCaseData).build();
     }
+
+    private void considerContestedMiniFormA(FinremCaseDetails finremCaseDetails,
+                                            String userAuthorisation) {
+
+        Optional<ContactDetailsWrapper> contactDetailsWrapper =
+                Optional.ofNullable(finremCaseDetails.getData().getContactDetailsWrapper());
```

Comment:
Looks like `getContactDetailsWrapper` will not return null.

Reply:
It doesn't.  But I think the null safe way that the Optional is used to getRespondentAddressHiddenFromApplicant and getApplicantAddressHiddenFromRespondent is nice and readable.  

(I refactored without Optional, and it's not as obvious what the code is doing)